scripts/india_census/primary_religion_data/preprocess.py

```diff
@@ -22,10 +22,44 @@
 from os import path
 import urllib.request
 from ..common.utils import title_case
-from ..common.generic_base import CensusGenericDataLoaderBase
 
-
-class CensusPrimaryReligiousDataLoader(CensusGenericDataLoaderBase):
+CENSUS_DATA_COLUMN_START = 7
+
+dcid_mapping = {}
```

Comment:
Can we use an initialized dict?

_RELIGION_DCID_MAPPING =  {
  "Hindu" : "dcs:Hinduism",
...
}

Reply:
/up/